lib/server.js

```diff
@@ -207,8 +208,12 @@ Server.prototype.stop = function (callback) {
     self._err = codeOrErr instanceof Error ? codeOrErr : self._err;
     callback(self._err);
   });
-
   proc.on("exit", _done);
+
+  // Kill it the "official" way.
   proc.kill();
+
+  // Kill it the unofficial way.
+  treeKill(proc.pid, "SIGKILL", _done);
```

Comment:
So, this utility actually shells to command line (linux, mac, windows, etc) and sends the proper signal. There's only one process coming back from `selenium-standalone`, so should be safe.